sap-concur/4.2/modules/ROOT/pages/index.adoc

```diff
@@ -114,30 +114,48 @@ Anypoint Studio provides two ways to add the connector to your Studio project: f
 
 This guide presents two of many use cases you might have for the Concur connector in your organization: getting a list of lists and retrieving quickexpenses. You may jump ahead and paste the code for the flows into the XML Editor in Studio after you download the Concur connector and create a global element to reference your Concur instance credentials.
 
-=== Retrieve a List of Lists
+=== Get List Items
 
 After creating a new project and a Concur global element:
 
 . Add a new flow element by dragging it from the palette and give a name to the flow, such as `getlists`.
 . Drag an HTTP Listener into your flow from the palette.
 . Double-click the HTTP Listener and click the green plus sign next to the Connector Configuration dropdown and set the Host to localhost, and the Port to 8081.
 . Click OK to close the properties window, then enter getlists as the value in the Path field in this HTTP listener's Basic Settings section.
-. Add a Concur connector to the new flow and select the Get list of lists operation, after selecting the Connector Configuration you desire.
+. Add a Concur connector to the new flow and select the *Get list items* operation, after selecting the Connector Configuration you desire.
```

Comment:
```suggestion
. Add a Concur connector to the new flow and select the *Get list items* operation after you select the Connector Configuration.
```

Reply:
It is unclear if selecting the connector configuration happens before adding the connector to the flow or after. If it happens before, it would be more straightforward to have this in a previous step. If it happens after, it would be more straightforward to phrase the step like this:

"Add a Concur connector to the new flow, select the connector configuration,  and select the *Get list items* operation.

Reply:
I see other issues with the original text. Maybe you can create a JIRA to track this for the future.